src/strategies/all-local.js

```diff
@@ -7,12 +7,10 @@ const {
 const { insertIfValue, insertIf } = require('../utils');
 
 module.exports = {
-  merge (srcs, { filterSubscriptions, logger }) {
+  merge (srcs, { filterSubscriptions, mergedSchemaTransforms = [], logger }) {
     const {
       typeDefs,
       resolvers,
-      extendTypes,
-      extendResolvers,
```

Comment:
modules.md -->
Modules can be either local type definitions and resolvers, or a [remote schema](./remote-module). Both can be freely combined and stitched together.

A module can expose the following values:
- name
- typeDefs
- resolvers
- extendTypes
- extendResolvers

Was it planned to remove these?

Reply:
Oh this is just an ES Lint issue. These variables were not being used :)